### ISSUE TITLE
No longer compatible with Windows 7-8.1

### ISSUE THREAD

**Issue by ghost** [SpotX-Official/SpotX#341]
### 🧐 Have you read the FAQ and viewed similar issues?

- [X] I have read the FAQ
- [X] I have checked older issues, open and closed
- [X] I did a clean install of SpotX as <a href="https://telegra.ph/SpotX-FAQ-09-19#1.-Clean-installation">described in the FAQ</a>

### Сountry of your account

Poland

### What is your Spotify Plan?

Free

### Do you have Spicetify installed?

No

### ℹ Computer information

```markdown
- Spotify: 1.2.6.863
- Windows: 7 SP1
- PowerShell: 5.1.14409.1029
```

### 📝 Description

1. Open Powershell
2. Insert the SpotX script and attempt to install
3. Error
I expected SpotX to install the latest compatible version of Spotify.

### 📸 Screenshots

![tLBj9z4qdY](https://user-images.githubusercontent.com/89755606/223566384-2b48e64d-2c14-442f-9deb-a972fc3d797f.png)

**Comment by amd64fox** [comment]
Yes, they started restricting the installer since 1.2.6

Did you check windows 8-8.1 or only windows 7?

**Comment by ghost** [comment]
> Yes, they started restricting the installer since 1.2.6
> 
> Did you check windows 8-8.1 or only windows 7?

I think Windows 8.1 grabbed 1.2.5 for me, I'll boot into my 8.1 install and let you know

EDIT: nevermind, Windows 8.1 also grabs 1.2.6. I suppose 8.0 also does that.
![8qkTKDEwIz](https://user-images.githubusercontent.com/89755606/223841386-da5a111d-5ddb-4718-bb21-883765a00d35.png)

**Comment by davidavida70** [comment]
Is there a way to disable the latest update installer? It's now impossible to use SpotX in windows 7.

**Comment by amd64fox** [comment]
@davidavida70 

I will try to fix it soon, we are waiting for the release of the new version

in the meantime, you can downgrade your version or use [ready-made builds](https://t.me/SpotxCommunity/1049) in our telegram

**Comment by ghost** [comment]
> @davidavida70
> 
> I will try to fix it soon, we are waiting for the release of the new version
> 
> in the meantime, you can downgrade your version or use [ready-made builds](https://t.me/SpotxCommunity/1049) in our telegram

Thanks for providing the ready-made builds, Spotify Web has been causing me nothing but issues

**Comment by amd64fox** [comment]
Unfortunately, it was not possible to adapt versions older than 1.2.5 for Windows 7-8.1, so the SpotX installer will simply limit the recommended version for Windows 7-8.1

**Comment by darkalexis1990** [comment]
can anyone patch the new 1.2.6 msi in order to install on 7/8.1?

**Comment by amd64fox** [comment]
above is a link to 1.2.5 there will be no difference with 1.2.6

**Comment by ghost** [comment]
> can anyone patch the new 1.2.6 msi in order to install on 7/8.1?

I think 1.2.6 is based on an Electron version which doesn't support Windows versions older than 10, so you'd have to patch the kernel first to support newer APIs.

**Comment by darkalexis1990** [comment]
hope someone do that :)

**Comment by darkalexis1990** [comment]
they patched 1.2.7  go on msfn for more info .
